apps/next/railway.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://railway.app/railway.schema.json",
+  "build": {
+    "builder": "NIXPACKS",
+    "buildCommand": "bun run build:next"
+  },
+  "deploy": {
+    "startCommand": "bun run start:next",
+    "healthcheckPath": "/",
+    "healthcheckTimeout": 100,
+    "restartPolicyType": "NEVER"
```

Comment:
reading the docs (https://docs.railway.app/reference/config-as-code#restart-policy-type) and it seems like ALWAYS with a restartPolicyMaxRetries set to something like 5 would be better

e.g. if a user triggers an error that causes the app to crash it should restart

Reply:
Done.